.github/workflows/main.yml

```diff
@@ -3,6 +3,8 @@ name: Sync Issue to Azure DevOps work item
 on:
   issues:
     types: [opened, edited, deleted, closed, reopened, labeled, unlabeled, assigned, unassigned]
+  issue_comment:
+    types: [created, edited]
```

Comment:
nice is that all that's required to turn on comments?

Reply:
Yeah, I did some research and I was wondering where he was getting "created" from. It's basically just GraphQL syntax